src/main/java/com/minecolonies/entity/ai/util/ItemStorage.java

```diff
@@ -0,0 +1,102 @@
+package com.minecolonies.entity.ai.util;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.item.ItemStack'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)